front-end/src/Navbar.jsx

```diff
@@ -48,22 +53,67 @@ function Navbar({ username, page }) {
         color: 'black'
     }
 
+    const fetchProfileData = async () => {
+        const accessToken = localStorage.getItem('accessToken');
+        if (!accessToken) {
+            console.error('Access token not available.');
```

Comment:
Same comment for `console.error`s. If you don't understand the reasoning please feel free to ask.

How are we handling this error? Ie., what happens if I click on the profile button and the `localStorage.getItem('accessToken')` fails? How do you handle it?